sharding/nodesCoordinator/indexHashedNodesCoordinatorLite_test.go

```diff
@@ -0,0 +1,89 @@
+package nodesCoordinator
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIndexHashedNodesCoordinator_SetNodesConfigFromValidatorsInfo(t *testing.T) {
+	t.Parallel()
+
+	arguments := createArguments()
+
+	ihgs, err := NewIndexHashedNodesCoordinator(arguments)
+	require.Nil(t, err)
+	epoch := uint32(1)
+
+	randomness := []byte("rand seed")
```

Comment:
Since `randomness` variable is only used in `ihgs.SetNodesConfigFromValidatorsInfo`, you could directly pass it there and avoid this initialization here